src/FlightDisplay/FlyView.qml

```diff
@@ -102,15 +102,6 @@ Item {
         guidedValueSlider:     _guidedValueSlider
     }
 
-    /*GuidedActionConfirm {
```

Comment:
Looks like this was commented out.  Why delete it?  We should try to do as few changes as possible to what is there.

Reply:
This was moved to another part of the code, this is the system arm slider which I had moved to the speed section of the UI.